packages/loader/src/gltf/extensions/MeshoptDecoder.ts

```diff
@@ -0,0 +1,203 @@
+    };
+
+    return worker;
+  }
+
+  function initWorkers(count: number) {
+    const source =
+      "var instance; console.time('init wasm');var ready = WebAssembly.instantiate(new Uint8Array([" +
+      new Uint8Array(unpack(wasm)) +
+      "]), {})" +
+      ".then(function(result) {instance = result.instance; instance.exports.__wasm_call_ctors();console.timeEnd('init wasm'); });" +
+      "self.onmessage = workerProcess;" +
+      decode.toString() +
+      workerProcess.toString();
+
+    const blob = new Blob([source], { type: "text/javascript" });
+    const url = URL.createObjectURL(blob);
+
+    for (let i = 0; i < count; ++i) {
+      workers[i] = createWorker(url);
+    }
+
+    URL.revokeObjectURL(url);
+  }
+
+  function decodeWorker(count: number, size: number, source: ArrayBuffer, mode, filter): Promise<Uint8Array> {
+    let worker = workers[0];
+
+    for (let i = 1; i < workers.length; ++i) {
+      if (workers[i].pending < worker.pending) {
+        worker = workers[i];
+      }
+    }
+
+    return new Promise(function (resolve, reject) {
+      const data = new Uint8Array(source);
+      const id = requestId++;
+
+      worker.pending += count;
+      worker.requests[id] = { resolve: resolve, reject: reject };
+      worker.object.postMessage(
+        {
+          id: id,
+          count: count,
+          size: size,
+          source: data,
+          mode: mode,
+          filter: filter
+        },
+        [data.buffer]
+      );
+    });
+  }
+
+  function workerProcess(event) {
+    ready.then(function () {
+      const data = event.data;
+      try {
+        const target = new Uint8Array(data.count * data.size);
+        decode(instance.exports[data.mode], target, data.count, data.size, data.source, instance.exports[data.filter]);
+        self.postMessage({ id: data.id, count: data.count, action: "resolve", value: target }, [target.buffer] as any);
+      } catch (error) {
+        self.postMessage({
+          id: data.id,
+          count: data.count,
+          action: "reject",
+          value: error
+        });
+      }
+    });
+  }
+
+  return {
+    ready: ready,
+    useWorkers: function (count) {
+      initWorkers(count);
+    },
+    decodeGltfBuffer: function (count: number, stride: number, source: Uint8Array, mode, filter): Promise<Uint8Array> {
+      if (workers.length > 0) {
+        return decodeWorker(count, stride, source, decoders[mode], filters[filter]);
+      }
+
+      return ready.then(function () {
+        const target = new Uint8Array(count * stride);
+        decode(instance.exports[decoders[mode]], target, count, stride, source, instance.exports[filters[filter]]);
+        return target;
+      });
+    },
+    destroy() {
+      for (let i = 0; i < workers.length; i++) {
+        workers[i].object.terminate();
+      }
+    }
+  };
+})();
+
+declare module "@galacean/engine-core" {
+  interface EngineConfiguration {
+    /** meshopt options. */
+    glTF?: {
```

Comment:
- Wrong comments?
- Please unify the Loader configuration naming style, such as KTX2